lib/cluster/app.rb

```diff
@@ -49,7 +49,15 @@ def self.find_existing
     def self.app_parameters
       stack = Stack.find_existing
       app_source = app_config[:app_source]
-      rds_arn = Cluster::RDS.writer_instance_arn
+      begin
+        rds_arn = Cluster::RDS.writer_instance_arn
+      rescue => e
+        unless e.message.include? "db_cluster_members"
+          raise
+        end
+        puts "rds missing or contains no instances"
+        rds_arn = ''
+      end
```

Comment:
when i'm using an ami-builder cluster i will delete the rds cloudformation stack and this try/catch prevents config updates from failing in that case.